src/engine/renderer/gl_shader.cpp

```diff
@@ -912,37 +911,25 @@ void GLShaderManager::buildPermutation( GLShader *shader, int macroIndex, int de
 	{
 		shader->BuildShaderCompileMacros( compileMacros );
 
-		if ( IsUnusedPermutation( compileMacros.c_str() ) )
+		if ( IsUnusedPermutation( compileMacros.c_str() ) ) {
```

Comment:
In these files the curly braces should go on a separate line. See https://wiki.unvanquished.net/wiki/Coding_convention#Quake_style

Reply:
Fixed.